content/6_Advanced/Parallel_Binary_Search.mdx

```diff
@@ -0,0 +1,140 @@
+---
+id: parallel-binary-search
+title: 'Parallel Binary Search'
+author: Muaath Alqarni
+description: 'Answer binary search queries offline'
+prerequisites:
+  - binary-search
+  - divide-and-conquer
+frequency: 0
+---
+
+
+## Resources
+
+<Resources>
+	<Resource source="CF" title="Parallel Binary Seach [tutorial]" url="45578" />
+	<Resource source="SPBook" title="Parallel Binary Seach" url="https://kostka.dev/sp/spbook.pdf#page=22" />
+</Resources>
+
+<FocusProblem problem="sam" />
+
+## Solution - New Roads Queries
+We can see that connectivity of the nodes is a monotonic function.
+
+Now let's use parallel binary search:
```

Comment:
give a quick summary on what pbs is and how one would use it
bc right now this is not different from a normal problem editorial